### DISCUSSION TITLE
Tips to find problems with long strings

### DISCUSSION THREAD

**Discussion by olaj** [fluentassertions/fluentassertions#1941] [Q&A] [answered]
I'm not sure if this is possible now or if this is a feature request.

Say just the simple

`result.Should().Be("Rhythmbox is an audio player for Linux that plays and organizes digital music. Inspired by Apple's.");`

and i get the result

`Expected result to be "Rhythmbox is an audio player for Linux that plays and organizes digital music." with a length of #, but "Rhythmbox is an audio player forLinux that plays and organizes digital music." has a length of ##, differs near "f" (index #).`

It would be so much easier if the result were presented like

"Rhythmbox is an audio player for Linux that plays and organizes digital music." - Expected with a length of #
"Rhythmbox is an audio player forLinux that plays and organizes digital music." - But differs near "f" (index #)

Or something like that, the point is that it would be awesome to see the problem just below the actual, now i usually copy it notepad and do this there.

**Comment by jnyrup** [comment]
There is certainly room for improvement here 🔧 

I like how xUnit substrings and aligns the string and even puts a pointer on the first difference.
```
Assert.Equal() Failure
                                 ↓ (pos 32)
Expected: ··· an audio player for Linux that plays and organizes digital m···
Actual:   ··· an audio player forLinux that plays and organizes digital mu···
                                 ↑ (pos 32)
```

**Comment by jnyrup** [comment, marked as answer]
I've created #2050 to track this work.
